webpack.config.js

```diff
@@ -0,0 +1,31 @@
+'use strict';
+
+let path = require('path');
+let SRC_DIR = path.join(__dirname, '/client/src/');
+let DIST_DIR = path.join(__dirname, '/client/dist/')
+
+module.exports = {
```

Comment:
Can you talk a little bit about your choice to use webpack in this project?

Reply:
Absolutely. I needed a transpilation tool because I would be using JSX and ES6 which aren’t widely supported yet. I could have just used standalone Babel, but since I’ve used the react-date-picker library before I knew I’d need to directly import its specific stylesheet in my JavaScript, which meant I’d need a CSS loader as well. Webpack can handle both of these concerns and is a tool I’m familiar with, which meant I could get up and running with it quickly which was important given the time constraint. It would also enable further CSS moduling to prevent name-space pollution issues if I wanted to grow the app and import component-specific styling. It also would open the door to code-splitting if I later found myself with a large, unwieldy bundle.js file.